README.md

```diff
@@ -97,14 +97,13 @@ this suite:
 
 ### Selenium Tests
 
-this suite has not been implemented yet.
+run these with `pytest --base-url http://localhost:3000 tests/ui`
```

Comment:
all of our other tests are distributed in the js code next to the stuff that they're testing. we have a /src/test place for making testing utilities, mocks, etc.

because of this, i don't think we're ever going to want to put anything else in the top level /tests folder, and having it wrap the single ui folder seems a little weird. can we rename the whole thing to /selenium or /ui-tests or something like that?

Reply:
I figured this would come up and just picked tests as that was standard for python. I also wasn't sure if we would employ python to do some other kind of testing like integration tests... so i split it up like that... thinking it may be `tests(or something else)/ui`,`tests/integration`, `tests/visual`

maybe just name the root folder to `pytest`?

Reply:
I based it off what mozilla had. https://github.com/mozilla/addons-server/tree/master/tests

Reply:
👍 for root folder `pytest`, if you think its prudent to make a subfolder below that go for it